openhands/runtime/client/runtime.py

```diff
@@ -225,34 +260,39 @@ def _init_container(
                 self.runtime_container_image,
                 command=(
                     f'/openhands/miniforge3/bin/mamba run --no-capture-output -n base '
-                    'PYTHONUNBUFFERED=1 poetry run '
-                    f'python -u -m openhands.runtime.client.client {self._port} '
-                    f'--working-dir {sandbox_workspace_dir} '
+                    f'poetry run '
+                    f'python -u -m openhands.runtime.client.client {self._container_port} '
+                    f'--working-dir "{sandbox_workspace_dir}" '
                     f'{plugin_arg}'
                     f'--username {"openhands" if self.config.run_as_openhands else "root"} '
                     f'--user-id {self.config.sandbox.user_id} '
                     f'{browsergym_arg}'
                 ),
                 network_mode=network_mode,
                 ports=port_mapping,
-                working_dir='/openhands/code/',
+                working_dir='/openhands/code/',  # do not change this!
                 name=self.container_name,
                 detach=True,
-                environment={'DEBUG': 'true'} if self.config.debug else None,
+                environment=environment,
                 volumes=volumes,
             )
             self.log_buffer = LogBuffer(container)
             logger.info(f'Container started. Server url: {self.api_url}')
             return container
         except Exception as e:
-            logger.error('Failed to start container')
+            logger.error(
+                f'VVVVV Instance {self.instance_id} FAILED to start container VVVVV'
```

Comment:
Not sure if we're doing these kinds of log decorators elsewhere, but it's typically something I try to avoid. Otherwise you end up with 100 of them because everyone thinks their error message is the most important 🙃  (ditto for `^^^^^^^^` below)

If we really want this, maybe we can add a parameter to logger like `prominent=True` so every caller gets the same treatment

Reply:
No problem, I'll remove those again.